### PR TITLE
Aizad/ chore: added reset button to account switcher and also reset function

### PR DESCRIPTION
**Pull request by aizad-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/javascript/app/pages/user/update_balance.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const BinarySocket          = require('../../base/socket');
 const formatMoney           = require('../../common/currency').formatMoney;
 const TopUpVirtualPopup     = require('../../pages/user/account/top_up_virtual/pop_up');
 const getPropertyValue      = require('../../../_common/utility').getPropertyValue;
+const createElement            = require('../../../_common/utility').createElement;
```
Comment (akmal-deriv) [inline src/javascript/app/pages/user/update_balance.js]:
```suggestion
const createElement         = require('../../../_common/utility').createElement;
```

File: src/javascript/app/pages/user/update_balance.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ const BinarySocket          = require('../../base/socket');
...
+const localize                 = require('../../../_common/localize').localize;
```
Comment (akmal-deriv) [inline src/javascript/app/pages/user/update_balance.js]: Hey, please make it aligned with previous lines

File: src/javascript/app/pages/user/update_balance.js [hunk 3]
```diff
@@ -18,14 +20,25 @@ const updateBalance = (response) => {
 
         const updateBalanceByAccountId = (account_id, updated_balance, account_currency) => {
             const el_balance_span = document.querySelector(`.account__switcher-balance-${account_id}`);
+            const reset_button      = createElement('button', { text: localize('Reset balance'), class: 'account__switcher-balance btn btn--secondary btn__small reset_btn' });
```
Comment (akmal-deriv) [inline src/javascript/app/pages/user/update_balance.js]: That's not aligned too

File: src/javascript/app/pages/user/update_balance.js [hunk 4]
```diff
@@ -18,14 +20,25 @@ const updateBalance = (response) => {
...
             if (el_balance_span) {
                 const display_balance = formatMoney(account_currency, updated_balance);
 
                 el_balance_span.innerHTML = display_balance;
-                
-                const is_virtual = /^VRT/.test(account_id);
-                const is_current = Client.get('loginid') === account_id;
+
+                // show reset button
+                if (updated_balance !== 10000 && is_virtual && is_current) {
```
Comment (akmal-deriv) [inline src/javascript/app/pages/user/update_balance.js:34]: What if the balance is bigger than 10000? Should it show reset button?

Comment (aizad-deriv) [reply src/javascript/app/pages/user/update_balance.js:34]: Yes. Only if balance is not equals to 10000 it will **not** show the reset button.

**Review by akmal-deriv — APPROVED** [review]

**Review by ameerul-deriv — APPROVED** [review]
LGTM

**Pull request merged**